src/Javis.jl

```diff
@@ -63,6 +63,8 @@ Create a video with a certain `width` and `height` in pixel.
 This also sets `CURRENT_VIDEO`.
 """
 function Video(width, height)
+    # some luxor functions need a drawing ;)
+    Drawing()
     video = Video(width, height, Dict{Symbol,Any}())
```

Comment:
cormullion is doing something similar to what we do with Action and Video such that we have `CURRENT_ACTION` as an example. All the drawing functions need a Drawing so if we call some drawing function before we call `javis` it will fail.
I think that this doesn't do any harm and makes it easier for the user.

Reply:
This also makes sense about why this would be required. Thanks for clarifying.